releases/release-4.0.13.md

```diff
@@ -0,0 +1,128 @@
+---
+title: TiDB 4.0.13 Release Notes
+---
+
+# TiDB 4.0.13 Release Notes
+
+Release date: May 28, 2021
+
+TiDB version: 4.0.13
+
+## Improvements
+
++ TiDB
+
+    - Skip reading mysql.stats_histograms if cached stats is up-to-date [#24352](https://github.com/pingcap/tidb/pull/24352)
+
++ TiFlash
+
+    - Automatically clean archive data to free up disk space [#1638](https://github.com/pingcap/tics/pull/1638)
+    - No release notes [#1625](https://github.com/pingcap/tics/pull/1625)
+
```

Comment:
+ PD
    - Metrics: let tso processing time not include consumption on the network [#3524](https://github.com/pingcap/pd/pull/3524)
    - Dashboard: update to v2021.03.12.1 [#3469](https://github.com/pingcap/pd/pull/3469)